docs/en-us/dev/user_doc/guide/installation/pseudo-cluster.md

```diff
@@ -35,6 +35,9 @@ echo "dolphinscheduler" | passwd --stdin dolphinscheduler
 sed -i '$adolphinscheduler  ALL=(ALL)  NOPASSWD: NOPASSWD: ALL' /etc/sudoers
 sed -i 's/Defaults    requirett/#Defaults    requirett/g' /etc/sudoers
 
+# Unzip the downloaded DolphinScheduler binary package to the specified directory, such as creating /opt/dolphinscheduler, and rename the unzipped file name
+tar -zxvf apache-dolphinscheduler-2.0.3-bin.tar.gz -C /opt/dolphinscheduler
+mv apache-dolphinscheduler-2.0.3-bin  dolphinscheduler-bin
 # Modify directory permissions and grant permissions for user you created above
 chown -R dolphinscheduler:dolphinscheduler dolphinscheduler-bin
```

Comment:
I think use in regexp mode is better, eg `chown -R dolphinscheduler:dolphinscheduler apache-dolphinscheduler-*-bin` WDYT

Reply:
OK，I'll change it

Reply:
> I think use in regexp mode is better, eg `chown -R dolphinscheduler:dolphinscheduler apache-dolphinscheduler-*-bin` WDYT

Modification complete